stories/footer.stories.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react'
-
 import Footer from '../app/components/footer'
 
 export default {
```

Comment:
Since the footer is suppose to go at the bottom of other components, and since there has been the problem with scrolling, I think the template of this story should have a <div style={height: 150vh}> above the footer, so that the footer would be below something that is bigger than the view port.  And you will have to wrap that div and the Footer in another div in order to return it. Or something like that. But create something above the footer. Maybe add a background color too.